backend/lcfs/web/api/allocation_agreement/repo.py

```diff
@@ -57,7 +58,7 @@ async def get_table_options(self, compliance_period: str) -> dict:
         provisions_of_the_act = (
             (await self.db.execute(provisions_select)).scalars().all()
         )
-        fuel_codes = (await self.db.execute(select(FuelCode))).scalars().all()
+        fuel_codes = (await self.db.execute(select(FuelCode))).unique().scalars().all()
```

Comment:
The .unique here is odd, FuelCode shouldn't contain any duplicates. What does this fix?

Reply:
Thank you! I removed  eager-loaded relationship.